File: 프로그래머스/0/120821. 배열 뒤집기/배열 뒤집기.java
class Solution {
    public int[] solution(int[] num_list) {
        int len = num_list.length;
        
        int[] answer = new int[len];
        
        for(int i = len - 1; i >= 0; i--){
            answer[len - i - 1] = num_list[i];
        }
        
        return answer;
    }
}

File: 프로그래머스/0/120821. 배열 뒤집기/배열 뒤집기.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int* solution(int num_list[], size_t num_list_len) {
    int* answer = (int*)malloc(num_list_len * sizeof(int));
    
    for(int i = 0; i < num_list_len; i++){
        answer[i] = num_list[num_list_len - i - 1];
    }
    
    return answer;
}